packages/cashier/src/pages/deposit/crypto-deposit/crypto-deposit.tsx

```diff
@@ -163,9 +168,17 @@ const CryptoDeposit = observer(() => {
                     </div>
                 ) : (
                     <>
-                        <Text as='p' align='center' line_height='m' size={isMobile() ? 'xs' : 's'}>
-                            {qrcode_header || header_note}
-                        </Text>
+                        {minimum_deposit ? (
+                            <AlertBanner
+                                className='crypto-third-party-alert'
+                                icon='IcAlertWarningDark'
+                                message={`A minimum deposit value of ${minimum_deposit} ${currency} is required. Otherwise, the funds will be lost and cannot be recovered.`}
```

Comment:
@aum-deriv not sure that Localize in AlertBanner can handle string with interpolations